### PR TITLE
taosdemo refactor split file

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: src/kit/taosdemo/src/demoFile.c [hunk 1]
```diff
@@ -0,0 +1,1760 @@
+/*
```
Comment (sangshuduo) [inline src/kit/taosdemo/src/demoFile.c]: as opposite to demoCommand.c, how about demoJsonOpt.c for more explicit?

Comment (sangshuduo) [reply src/kit/taosdemo/src/demoFile.c]: demoCommand could be something like demoCommandOpt

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by afwerar — APPROVED** [review]
LGTM

**Pull request merged**